designer/server/src/plugins/routes/api.test.js

```diff
@@ -169,10 +161,13 @@ describe('Server API', () => {
     }
 
     // When
-    const result = await server.inject<{ err: Error }>(options)
+    const result =
+      /** @type {import('@hapi/hapi').ServerInjectResponse<{ err: Error }>}) */ (
+        await server.inject(options)
+      )
 
     // Then
-    expect(result.statusCode).toBe(401)
+    expect(result.statusCode).toBe(500)
```

Comment:
No idea why it would return Unauthorised for a generic error. 500 is what we should return... I did a check and I couldn't find that the 401 is used anywhere.